weatherman.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+from code_files.weather_parser import WeatherParser
+from code_files.weather_reporter import WeatherReporter
+from code_files.report_calculator import ReportCalculator
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("extract_to", type=str)
+    parser.add_argument("-e", "--extremes", type=int)
```

Comment:
Let's also come up with more useful name args e.g `-e` & `--yearly_report` instead of `--extremes`

Reply:
[Meaning argument names](https://github.com/ttqureshi/WeatherMan/pull/1/commits/b6d935340b0b0850b476f6ec2ed11e67d57d5fe8)